contracts/mock/util/AddressLibMock.sol

```diff
@@ -31,10 +31,6 @@ contract AddressLibMock {
         self.requireNotEmpty(REQUIRE_NOT_EMPTY_ERROR_MESSAGE);
     }
 
-    function requireEmpty(address self) external pure {
-        self.requireEmpty(REQUIRE_EMPTY_ERROR_MESSAGE);
-    }
-
```

Comment:
Need to readd as it is now being used by Chainlink

Reply:
Changed ChainLinkAggregator to use a require with isEmpty() instead